manager/manager.go

```diff
@@ -69,73 +72,95 @@ func NewTransport(dc dispatch.DispatcherConfig) http.RoundTripper {
 // chrysom client listener
 func (m *Manager) Update(items []argus.Item) {
 	recentMap := make(map[string]model.Norn)
-	newNorns := []nornDispatcher{}
-	oldNorns := []dispatch.D{}
+	newNorns := []filterDispatcher{}
+	oldNorns := []dispatch.F{}
 
 	transport := NewTransport(m.dispatcherConfig)
+	sender := &http.Client{
+		Transport: transport,
+	}
+	var dispatcher dispatch.D
 
 	for _, item := range items {
 		id := item.Identifier
 		norn, err := model.ConvertItemToNorn(item)
 		if err != nil {
 			log.WithPrefix(m.logger, level.Key(), level.ErrorValue()).Log(logging.MessageKey(), "failed to convert Item to Norn", "item", item)
 		}
-		if _, ok := m.nornsDispatch[id]; ok {
+		if _, ok := m.nornFilter[id]; ok {
 			recentMap[id] = norn
 		} else {
-			dispatcher, err := dispatch.NewDispatcher(m.dispatcherConfig, norn, transport)
+
+			if (norn.Destination.AWSConfig) == (model.AWSConfig{}) {
+				dispatcher, err = dispatch.NewHttpDispatcher(m.dispatcherConfig, norn, sender)
+				if err != nil {
+					m.logger.Log(level.Key(), level.ErrorValue(), logging.MessageKey(), err.Error)
+				}
+			} else {
+				dispatcher, err = dispatch.NewSqsDispatcher(m.dispatcherConfig, norn, transport)
+				if err != nil {
+					m.logger.Log(level.Key(), level.ErrorValue(), logging.MessageKey(), err.Error)
+				}
+			}
+
+			filter, err := dispatch.NewFilter(m.filterConfig, dispatcher, norn, sender)
+			if err != nil {
+				m.logger.Log(level.Key(), level.ErrorValue(), logging.MessageKey(), err.Error)
+			}
 			if err == nil {
-				newNorns = append(newNorns, nornDispatcher{norn: norn, dispatcher: dispatcher})
+				newNorns = append(newNorns, filterDispatcher{norn: norn, filter: filter, dispatcher: dispatcher})
 				recentMap[id] = norn
 			} else {
 				m.logger.Log(level.Key(), level.ErrorValue(), logging.MessageKey(), err.Error)
 			}
+
 		}
 	}
 
-	for i, norndis := range m.nornsDispatch {
-		if val, ok := recentMap[i]; !ok {
-			oldNorns = append(oldNorns, norndis.dispatcher)
-			delete(m.nornsDispatch, i)
+	for i, filterdis := range m.nornFilter {
+		if norn, ok := recentMap[i]; !ok {
+			oldNorns = append(oldNorns, filterdis.filter)
+			delete(m.nornFilter, i)
 		} else {
-			norndis.dispatcher.Update(val)
+			filterdis.dispatcher.Update(norn)
+			filterdis.filter.Update(norn)
 		}
 	}
 
-	for _, norndis := range newNorns {
+	for _, filterdis := range newNorns {
```

Comment:
I'm ignoring this function since it will need to be updated to account for two maps in the struct.  If you're not sure how to update it to account for that, we can talk more about it over slack.